lib/dndstrings-captions.sty

```diff
@@ -0,0 +1,26 @@
+% Italian captions
+\addto\captionsitalian{%
+  \def\armorclassname{Classe Armatura}
+  \def\hitpointsname{Punti Ferita}
+  \def\speedname{Velocità}
```

Comment:
LaTex does not necessarily handle accents in text well. à was garbled when I tested it. This accent should be entered as \\`a.
<https://tex.stackexchange.com/tags/accents/info>

Reply:
You're right, I forgot about that since I usually compile these with LuaLaTeX. Fixed now.